### ISSUE TITLE
Chrome: Cannot scroll in Monaco editors when browser page search is open

### ISSUE THREAD

**Issue by slimsag** [sourcegraph/sourcegraph#8526]
1. Visit https://sourcegraph.com/site-admin/global-settings
2. Use cmd+f to open your browser page search
3. Try to scroll in the Monaco editor -- it doesn't work and instead scrolls the outer page.

This confused me because Chrome's browser search is relatively non-intrusive, so I didn't realize I had it open and was confused why I couldn't scroll in the settings. I ended up using the click-n-drag to scroll feature which was painful.

**Comment by felixfbecker** [comment]
I cannot reproduce this - @slimsag does it still happen for you?

**Comment by slimsag** [comment]
Yes, I can still reproduce this on the latest version of Chrome.

- Load page, scrolling with mouse over editor works fine.
- Open browser page search, scrolling no longer scrolls the Monaco editor and instead scrolls outer page.
- Close browser page search, scrolling in Monaco editor remains broken and you have to use the click-n-drag feature instead.

http://www.giphy.com/gifs/J5M5smZhUKmLOU73Zz

**Comment by felixfbecker** [comment]
Thanks for trying again!
- Which OS are you using?
- Are you using a trackpad or a mouse?
- Is your cursor over the Monaco editor when you attempt to scroll?
- Does it work again when you click inside (focus) the Monaco editor?
